releases/release-6.5.2.md

```diff
@@ -0,0 +1,135 @@
+
++ TiKV
+
+    - 修复 tikv 解析 cgroup path 没有正确处理 ":" 符号的问题 [#14535](https://github.com/tikv/tikv/pull/14535) @[SpadeA-Tang](https://github.com/SpadeA-Tang)
+
++ PD
+
+    - (dup): release-6.1.4.md > Bug 修复> PD - 修复 PD 可能会非预期地向 Region 添加多个 Learner 的问题 [#5786](https://github.com/tikv/pd/issues/5786) @[HunDunDM](https://github.com/HunDunDM)
+    - (dup): release-7.0.0.md > 错误修复> PD - 修复了切换 Placement Rule 时可能存在的 leader 分布不均衡的问题 [#6195](https://github.com/tikv/pd/issues/6195) @[bufferflies](https://github.com/bufferflies)
+
++ TiFlash
+
+    - (dup): release-6.1.6.md > Bug 修复> TiFlash - 修复 TiFlash 无法识别生成列的问题 [#6801](https://github.com/pingcap/tiflash/issues/6801) @[guo-shaoge](https://github.com/guo-shaoge)
+    - (dup): release-7.0.0.md > 错误修复> TiFlash - 修复了 Decimal 除法在某些情况下最后一位未进位的问题 [#7022](https://github.com/pingcap/tiflash/issues/7022) @[LittleFall](https://github.com/LittleFall)
+    - (dup): release-7.0.0.md > 错误修复> TiFlash - 修复了 Decimal 转换在某些情况下进位错误的问题 [#6994](https://github.com/pingcap/tiflash/issues/6994) @[windtalker](https://github.com/windtalker)
+    - (dup): release-7.0.0.md > 错误修复> TiFlash - 修复了开启 new collation 后 TopN/Sort 算子结果可能出错的问题 [#6807](https://github.com/pingcap/tiflash/issues/6807) @[xzhangxian1008](https://github.com/xzhangxian1008)
+    - Fix the issue of TiFlash process failures due to TiCDC incompatibility [#7212](https://github.com/pingcap/tiflash/issues/7212) @[hongyunyan](https://github.com/hongyunyan)
+
++ Tools
+
+    + Backup & Restore (BR)
+
+        - (dup): release-6.6.0.md > 错误修复> Tools> Backup & Restore (BR) - 修复当 TiDB 集群不存在 PITR 备份任务时，`resolve lock` 频率过高的问题 [#40759](https://github.com/pingcap/tidb/issues/40759) @[joccau](https://github.com/joccau)
+        - (dup): release-7.0.0.md > 错误修复> Tools> Backup & Restore (BR) - 修复了在 PITR 恢复过程中等待 split Region 重试的时间不足的问题 [#42001](https://github.com/pingcap/tidb/issues/42001) @[joccau](https://github.com/joccau)
+
+    + TiCDC
+
+
+        - 修复同步到对象存储时 partition 分隔符不生效问题 [#8581](https://github.com/pingcap/tiflow/issues/8581)
+        - 修复同步到对象存储场景下，发生表调度可能导致数据丢失问题  [#8256](https://github.com/pingcap/tiflow/issues/8256)
+        - 修复不可重入的 DDL 带来卡住的问题 [#8662](https://github.com/pingcap/tiflow/issues/8662)
+        - 修复同步到对象存储场景下，TiCDC 做扩缩容导致数据丢失问题 [#8666](https://github.com/pingcap/tiflow/issues/8666)
+        - 修复利用 cgroup 在部分场景内存限制不生效问题  [#8588](https://github.com/pingcap/tiflow/issues/8588)
+        - 修复同步到对象存储场景下，TiCDC 做扩缩容导致数据丢失问题 [#8666](https://github.com/pingcap/tiflow/issues/8666)
+        - 修复Redo log 在 apply 时，特殊场景可能出现数据丢失的问题 [#8591](https://github.com/pingcap/tiflow/issues/8591)
+        - 采用 gomemlimit 防止 TiCDC 出现 OOM 问题[#8675](https://github.com/pingcap/tiflow/issues/8675)
+        - 修复同步到 Kafka( canal-json) 场景下，同步 float 类型字段出现精度丢失问题 [#8490](https://github.com/pingcap/tiflow/issues/8490)
+        - 采用 multi-statement 的方式优化批量做 update 场景下的同步性能   [#8057](https://github.com/pingcap/tiflow/issues/8057)
```

Comment:
```suggestion
        - Fix the issue that the partition delimiter does not work when TiCDC replicates data to object storage [#8581](https://github.com/pingcap/tiflow/issues/8581) @[CharlesCheung96](https://github.com/CharlesCheung96) @[hi-rustin](https://github.com/hi-rustin)
        - Fix the issue that table scheduling might cause data loss when TiCDC replicates data to object storage [#8256](https://github.com/pingcap/tiflow/issues/8256) @[zhaoxinyu](https://github.com/zhaoxinyu)
        - Fix the issue that non-reentrant DDL statements get stuck [#8662](https://github.com/pingcap/tiflow/issues/8662) @[hicqu](https://github.com/hicqu)
        - Fix the issue that TiCDC scaling might cause data loss when TiCDC replicates data to object storage [#8666](https://github.com/pingcap/tiflow/issues/8666) @[CharlesCheung96](https://github.com/CharlesCheung96)
        - Fix the issue that the memory limit of cgroup does not take effect in some scenarios  [#8588](https://github.com/pingcap/tiflow/issues/8588) @[amyangfei](https://github.com/amyangfei)
        - Fix the issue that data loss might occur in special cases during the apply of Redo log [#8591](https://github.com/pingcap/tiflow/issues/8591) @[CharlesCheung96](https://github.com/CharlesCheung96)
        - Introduce gomemlimit to prevent TiCDC from experiencing OOM issues [#8675](https://github.com/pingcap/tiflow/issues/8675) @[amyangfei](https://github.com/amyangfei)
        - Use the multi-statement approach to optimize the replication performance in scenarios involving batch execution of `UPDATE` statements [#8057](https://github.com/pingcap/tiflow/issues/8057) @[amyangfei](https://github.com/amyangfei)
```